packages/angular/src/utils/versions.ts

```diff
@@ -2,7 +2,7 @@ export const nxVersion = '*';
 export const angularVersion = '^8.0.0';
 export const angularDevkitVersion = '^0.800.0';
 export const angularJsVersion = '1.6.6';
-export const ngrxVersion = '7.2.0';
+export const ngrxVersion = '8.1.0';
 export const ngrxStoreFreezeVersion = '0.2.4';
```

Comment:
```suggestion
```

I believe we can remove this now as it is no longer used anywhere.

Reply:
Yes, it can be removed :)